### PR TITLE
update titanic

### PR DESCRIPTION
**Pull request by LittleLittleCloud**

### CODE REVIEW

File: machine-learning/E2E-Binary Classification with Titanic Dataset.ipynb [hunk 1]
```diff
@@ -1,195 +1,364 @@
...
\ No newline at end of file
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Titanic tutorial using proposed AutoML API\n",
+        "### Disclaimer: This is for demo and private preview purposes only.\n",
+        "### Disclaimer: This notebook uses features that may be broken or unsupported at any time.\n",
```
Comment (JakeRadMSFT) [inline machine-learning/E2E-Binary Classification with Titanic Dataset.ipynb:9]: We can get rid of these comments now.

File: machine-learning/E2E-Binary Classification with Titanic Dataset.ipynb [hunk 2]
```diff
@@ -1,195 +1,364 @@
...
-      "name": "C#",
-      "pygments_lexer": "csharp",
-      "version": "8.0"
-    }
-  },
-  "nbformat": 4,
-  "nbformat_minor": 4
-}
\ No newline at end of file
+{
```
Comment (JakeRadMSFT) [inline machine-learning/E2E-Binary Classification with Titanic Dataset.ipynb:1]: Can you rename this to -  REF-Kaggle with Titanic Dataset.ipynb

**Comment by JakeRadMSFT** [comment]
Do we want the submission.csv checked in?

**Comment by LittleLittleCloud** [comment]
> Do we want the submission.csv checked in?

Probabaly, so we can refer the kaggle score using that submission.

**Review by JakeRadMSFT — APPROVED** [review]

**Review by JakeRadMSFT — APPROVED** [review]

**Pull request merged**